src/createServer.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
-// const express = require('express');
+const cors = require('cors');
+const express = require('express');
+const usersRouter = require('./routes/users');
+const expensesRouter = require('./routes/expenses');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  app.use(cors());
+  app.use('/users', usersRouter.router);
+  app.use('/expenses', expensesRouter.router);
```

Comment:
You can use middleware here and don't repeat in routes
```suggestion
  app.use('/users', express.json(), usersRouter.router);
  app.use('/expenses',  express.json(), expensesRouter.router);
```

Reply:
Agree, that way will be easier as for me

Reply:
But it's not critical